tests/cpu/test_ckf_sparse_tracks_telescope.cpp

```diff
@@ -123,9 +126,10 @@ TEST_P(CkfSparseTrackTelescopeTests, Run) {
     // Finding algorithm configuration
     typename traccc::finding_config cfg;
     cfg.ptc_hypothesis = ptc;
-    cfg.chi2_max = 30.f;
+    cfg.chi2_max = 200.f;
```

Comment:
After I set the B field to z-axis and Telescope planes set along the x-axis, chi2 cut value of 30 for the CKF selection did not work for a few measurements :/ 

I believe the cut value of 30 should be OK in most cases from the distribution of chi2 of the Kalman Fitter (See the plot below). The change to 200 is just to cover the outliers which happen very rarely

<img src="https://github.com/user-attachments/assets/b1690a84-b8e0-4542-97cd-b180bd21b0c9" width="400" height="300" />